src/extension/commands/index.ts

```diff
@@ -18,6 +18,16 @@ export function copyCellToClipboard (cell: NotebookCell) {
   return window.showInformationMessage('Copied cell to clipboard!')
 }
 
+export function stopBackgroundTask (cell: NotebookCell) {
+  const terminal = getTerminalByCell(cell)
+  if (!terminal) {
+    return window.showWarningMessage('Couldn\'t find terminal! Was it already closed?')
```

Comment:
My inner PTSD says that this lines does the same as line `11` and one could put this into a function and re-use it. I will leave it up to you.